components/automate-ui/src/app/modules/infra-proxy/policy-groups/policy-groups.component.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Component,
+  Input,
+  OnInit,
+  OnDestroy,
+  EventEmitter,
+  Output } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { combineLatest, Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { isNil } from 'lodash/fp';
+import { LayoutFacadeService, Sidebar } from 'app/entities/layout/layout.facade';
+import { NgrxStateAtom } from 'app/ngrx.reducers';
+import { EntityStatus } from 'app/entities/entities';
+import { GetPolicyGroups } from 'app/entities/policy-files/policy-file.action';
+import { PolicyFile } from 'app/entities/policy-files/policy-file.model';
+import {
+  getGroupsStatus,
+  policyFile
+} from 'app/entities/policy-files/policy-group.selectors';
+
+@Component({
+  selector: 'app-policy-groups',
+  templateUrl: './policy-groups.component.html',
+  styleUrls: ['./policy-groups.component.scss']
+})
+export class PolicyGroupsComponent implements OnInit, OnDestroy {
+  @Input() serverId: string;
+  @Input() orgId: string;
+  @Output() resetKeyRedirection = new EventEmitter<boolean>();
+
+  private isDestroyed = new Subject<boolean>();
+  public policyGroups: PolicyFile[] = [];
+  public policyGroupsListLoading = true;
+  public authFailure = false;
+
+  constructor(
+    private store: Store<NgrxStateAtom>,
+    private layoutFacade: LayoutFacadeService
+  ) { }
+
+  ngOnInit(): void {
+    this.layoutFacade.showSidebar(Sidebar.Infrastructure);
+
+    this.getPolicyGropus();
+
+    combineLatest([
+      this.store.select(getGroupsStatus),
+      this.store.select(policyFile)
+    ]).pipe(takeUntil(this.isDestroyed))
+    .subscribe(([ getPolicyFilesSt, allPolicyFilesState]) => {
+      if (getPolicyFilesSt === EntityStatus.loadingSuccess && !isNil(allPolicyFilesState)) {
+        this.policyGroups = allPolicyFilesState.policyFile;
```

Comment:
we are getting policyfiles so it should name as policyfile

Reply:
Updated allPolicyFilesState to policyFilesState. 
Thank you.